actions/llm-action-error-reporter/action.yml

```diff
@@ -76,16 +76,36 @@ runs:
         # prevents access to any other action and workflow files
         sparse-checkout: |
           actions/llm-action-error-reporter/log-analyze-prompt.md
+          actions/llm-action-error-reporter/update_pr_comment.sh
+          actions/llm-action-error-reporter/trim_logs.sh
         sparse-checkout-cone-mode: false
         fetch-depth: 1
 
+    - name: Prepare logs
+      if:
+        ${{ env.SKIP_ACTION == 'false' && inputs.parent-workflow-conclusion ==
+        'failure' }}
+      shell: bash
+      env:
+        GH_TOKEN: ${{ inputs.gh-token }}
+        LINE_LIMIT: ${{ inputs.log-lines-limit }}
+      run: |
+        echo "Analyzing run ${{ github.event.workflow_run.id }}"
+        LOG_RAW=$(gh run view ${{ github.event.workflow_run.id }} --log-failed)
+        echo "$LOG_RAW" > rawlog.log
+
+        # trim logs and outputs to runlog.log
+        ./error_reporter_actions/actions/llm-action-error-reporter/trim_logs.sh
```

Comment:
Is this path correct? Here's an example of how we can refer to the action path dynamically: 

https://github.com/smartcontractkit/.github/blob/a40ee26a986e00fbef228ded7e7da9bfed64dd2c/actions/cicd-build-publish-charts/action.yml#L93

Reply:
@chainchad
The path is correct and uses the sparse-checkout path ( tested with rev in the test repository .)
Changed to go with your suggestion as these scripts aren't meant to be customizable like the prompt file.